lib/docusign_rest/client.rb

```diff
@@ -573,28 +573,29 @@ def initialize_net_http_multipart_post_request(uri, post_body, file_params, head
 
     # Public: creates an envelope from a document directly without a template
     #
-    # file_io       - Optional: an opened file stream of data (if you don't
-    #                 want to save the file to the file system as an incremental
-    #                 step)
-    # file_path     - Required if you don't provide a file_io stream, this is
-    #                 the local path of the file you wish to upload. Absolute
-    #                 paths recommended.
-    # file_name     - The name you want to give to the file you are uploading
-    # content_type  - (for the request body) application/json is what DocuSign
-    #                 is expecting
-    # email_subject - (Optional) short subject line for the email
-    # email_body    - (Optional) custom text that will be injected into the
-    #                 DocuSign generated email
-    # signers       - A hash of users who should receive the document and need
-    #                 to sign it. More info about the options available for
-    #                 this method are documented above it's method definition.
-    # status        - Options include: 'sent', 'created', 'voided' and determine
-    #                 if the envelope is sent out immediately or stored for
-    #                 sending at a later time
-    # customFields  - (Optional) A hash of listCustomFields and textCustomFields.
-    #                 Each contains an array of corresponding customField hashes.
-    #                 For details, please see: http://bit.ly/1FnmRJx
-    # headers       - Allows a client to pass in some
+    # file_io        - Optional: an opened file stream of data (if you don't
+    #                  want to save the file to the file system as an incremental
+    #                  step)
+    # file_path      - Required if you don't provide a file_io stream, this is
+    #                  the local path of the file you wish to upload. Absolute
+    #                  paths recommended.
+    # file_name      - The name you want to give to the file you are uploading
+    # content_type   - (for the request body) application/json is what DocuSign
+    #                  is expecting
+    # email/subject  - (Optional) short subject line for the email
+    # email/body     - (Optional) custom text that will be injected into the
+    #                  DocuSign generated email
+    # email/reply_to - Sets the Reply-To email used for the envelope
```

Comment:
Would it make sense to prepend `(Optional) `, so as to maintain consistency with previous `email` options documentation?